virt/virt-4-10-release-notes.adoc

```diff
@@ -54,6 +54,9 @@ The SVVP Certification applies to:
 //CNV-14192 Update launchers using live-migration - default/opt out
 * {VirtProductName} workloads, such as `virt-launcher` pods, now automatically update if they support live migration. You can xref:../virt/upgrading-virt.adoc#configuring-workload-updates_upgrading-virt[configure workload update strategies] or opt out of future automatic updates by editing the `HyperConverged` custom resource.
 
+//CNV-16637 SNO availability
+* You can now use {VirtProductName} with xref:../virt/install/preparing-cluster-for-virt.adoc#virt-single-node-cluster_preparing-cluster-for-virt[single node clusters], also known as Single Node OpenShift (SNO). These clusters are not configured for high-availability operation, which results in significant changes to {VirtProductName} behavior.
```

Comment:
I think the last sentence might be better as a note (or even a warning?). I'd add it to the end of the RN like this:

```
* You can now use...
+
[NOTE]
====
Single node clusters are not...
====
```